src/McpDotNet.Extensions.AI/AIFunctionUtilities.cs

```diff
@@ -1,26 +1,49 @@
 ﻿using McpDotNet.Protocol.Types;
 using System.Text.Json;
+using System.Text.Json.Serialization;
```

Comment:
FWIW, I'm about to submit a PR that guts this assembly, as it's no longer relevant once the M.E.AI functionality is integrated into the core library. I can fix things up to react to your changes, though.